src/lib/cache/abstract_cache_impl.hpp

```diff
@@ -64,6 +66,8 @@ class AbstractCacheImpl {
   virtual ErasedIterator begin() = 0;
   virtual ErasedIterator end() = 0;
 
+  size_t frequency(const Key& key) { Fail("Cache implementation does not suppport frequency()."); }
```

Comment:
If only one cache implementation supports this, why don't we add it just there?

Reply:
You mean as a virtual method which is only implemented in GDFS?

Reply:
No, as a non-virtual method. Why should it be virtual if there is exactly one implementation.

Reply:
That would require us to always cast the PQP cache (`using SQLPhysicalPlanCache = Cache<std::shared_ptr<AbstractOperator>, std::string>;`) to GDFS, doesn't it?

Reply:
Yes, but I consider that a good thing.

Reply:
I don't think so, but ok. Will change.

Reply:
Addressed.